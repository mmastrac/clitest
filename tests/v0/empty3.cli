#!/usr/bin/env clitest --v0

# Empty script
