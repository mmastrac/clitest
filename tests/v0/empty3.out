Running <root>/empty3.cli ...

<root>/empty3.cli PASSED
